### ISSUE TITLE
Role fails to install JRE on Ubuntu 14.04

### ISSUE THREAD

**Issue by danielkza** [kevincoakley/ansible-role-zeppelin#4]
It manually skips the package installation step:

``` yaml
- name: Download and install the required packages (apt)
  apt:
    name: "{{ item }}"
    state: present
  with_items: "{{ zeppelin_packages }}"
  when: ansible_os_family == "Debian" and ansible_distribution_version != '14.04'
```

Which is problematic, since it means the role will complete without actually succeeding. It should actually verify that Java is installed through some other means (such as having a custom java home set), or attempt to do the install in 14.04 anyway - it will succeed if a custom repository is installed, and fail otherwise.

**Comment by kevincoakley** [comment]
@danielkza @smiller171 The reason why I didn't include the install for Java 1.8 on Ubuntu 14.04 is (1) I am not a heavy Ubuntu user and (2) Canonical doesn't include 1.8 for 14.04, so I don't know what the best practice is for installing 1.8 and which PPA should be used. 

I know there are PPAs for both Oracle Java and OpenJDK and both seem to have their own issues (Oracle being proprietary and OpenJDK being out of date). 

Here are possible solutions:
1. Write an Ansible check for Java 1.8 and fail if it isn't installed
2. Merge #5 
3. Remove Java completely from this role and require another role that installs Java

Let me know what you think. It is highly unlikely that I would ever use Ubuntu 14.04 so which solution you choose will not affect me.

**Comment by danielkza** [comment]
@kevincoakley I'd personally prefer option 1, since the PPAs do have issues, as you mentioned. It's enough for me to have the role fail when a sufficient version of Java is missing. But what about another option: installing Java 7 on 14.04? It seems to be supported according to the [documentation](https://zeppelin.apache.org/docs/0.6.1/install/install.html).

**Comment by kevincoakley** [comment]
@danielkza The data scientists that I work with all use Java 8 so I never bothered to look at the documentation to find the minimum Java requirements. Using Java 7 with Ubuntu 14.04 does seem like the most logical solution. I will take a look at added Java 7 + Ubuntu 14.04 and adding a check to verify Java is installed.

**Comment by smiller171** [comment]
I'll go ahead and close my PR then

**Comment by danielkza** [comment]
@kevincoakley Would you like me to work on it? I have a bit of free time today.

**Comment by kevincoakley** [comment]
@danielkza Sure, I have errands to run in the morning, so I won't be able to look at this until this afternoon. Thank you!

**Comment by kevincoakley** [comment]
Merged pull request #6
